latch_cli/main.py

```diff
@@ -203,31 +206,76 @@ def init(
 
 
 @main.command("cp")
-@click.argument("source_file", nargs=1)
-@click.argument("destination_file", nargs=1)
-def cp(source_file: str, destination_file: str):
+@click.argument("src", nargs=1)
+@click.argument("dest", nargs=1)
+@click.option(
+    "--max-concurrent-files",
+    "--mcf",
+    help="Maximum number of files to copy concurrently.",
+    type=int,
+    default=4,
+    show_default=True,
+)
+@click.option(
+    "--progress",
+    help="Type of progress information to show while copying",
+    type=click.Choice(list(Progress._member_names_), case_sensitive=False),
```

Comment:
We might try to do something spicy with enums so that we do not have to handle every enum parameter. https://github.com/pallets/click/issues/605. Not sure this is better tho.

Reply:
Click maintainers seem like assholes